### PR TITLE
various fixes ...

### PR DESCRIPTION
**Pull request by braunms**
- fixed font size for cinematic stuff (font size 0 lead to infinite loop when calling mmSetGUIVisible(false) in project file)
- improved UX for scaling and font menu
- fixed some stuff in OverlayRenderer
- fixed bugs in logconsole auto completion
- added File_Any flag to FilePathParam to allow selection of files and directories in the same parameter
- ModernTrisoupRenderer: fixed mesh color, reset gl state of polygon mode
- Fixed GUI visibility toggling

### CODE REVIEW

File: frontend/services/gui/src/GUIManager.cpp [hunk 1]
```diff
@@ -1149,18 +1149,23 @@ void GUIManager::draw_menu() {
             float scale = megamol::gui::gui_scaling.Get();
             if (ImGui::RadioButton("100%", (scale == 1.0f))) {
                 this->SetScale(1.0f);
+                ImGui::CloseCurrentPopup();
```
Comment (reinago) [inline frontend/services/gui/src/GUIManager.cpp]: I am not really sure I want that. Good question actually.

Comment (braunms) [reply frontend/services/gui/src/GUIManager.cpp]: Updated slightly in following commit to fit UX of other menus.

**Comment by reinago** [comment]
I just looked at this and using the cinematic example I opened the log console, used mmHelp (using autocomplete) and could not scroll to the bottom anymore...

**Review by moritz-h — DISMISSED** [review]

**Review by moritz-h — APPROVED** [review]

**Pull request merged**